index.js

```diff
@@ -0,0 +1,72 @@
+const defaultOptions = {
+    styles: '*:focus  { outline: none !important; }',
+    stylesheetTarget: document.head,
+};
+
+const acceptedKeys = { Tab: 9, ArrowLeft: 37, ArrowTop: 38, ArrowRight: 39, ArrowDown: 40 };
```

Comment:
Not sure about the convention we should use here, but, for constants, I've already seen fully uppercased variables being used. 

@satazor could you to provide input here? Thanks. 🙏

Reply:
Yea perhaps put them uppercase and snake case: `DEFAULT_OPTIONS`, `ACCEPTED_KEYS`.